### PR TITLE
Tribute page Code Review 

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project I did:

- [ ] Add HTML file
- [ ] Add CSS file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script>
+    <!-- <h1>Hello World! We're software developers</h1> -->
```
Comment (tumainimaganiko) [inline index.html:12]: Please do not leave unused code as a comment. Unused code as comments makes your code appear not clean

File: index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <div id="main">
+      <h1 id="title">Mahatma Gandhi</h1>
+    </div>
+    <div id="img-div">
```
Comment (tumainimaganiko) [inline index.html:16]: I kindly recommend you use HTML5 semantic tags (e.g., header, nav) over divs

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#image {
```
Comment (tumainimaganiko) [inline styles.css:1]: I recommend you specify the width and height of your image, it improves the appearance of your image

**Review by tumainimaganiko — COMMENTED** [review]
Hello
 You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!


Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅
- The PR title and description are professional and descriptive ✅
- The README is well documented ✅

# Changes required ♻️ 
Please check my comments below and make all required changes.

Submit for a re-review when you're done.
Thanks 😁

**Review by Shisui6 — APPROVED** [review]
Hi @tumainimaganiko, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Great summary ✅
- Good use of Github inline comments ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

- [Optional] Good job adding emojis and gifs to your summary👏. However, it would also be nice if you added them to your inline comments as well. This makes your entire review feel more human friendly🫡

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**